package.json

```diff
@@ -66,6 +66,7 @@
     "codemirror": "~5.47.0",
     "connected-react-router": "~6.7.0",
     "create-react-context": "~0.2.3",
+    "d3": "^5.5.0",
```

Comment:
the version of d3 being required by patternfly (and thus used by us) is 3.5.17. 5 is too new.

Reply:
@himdel Same version is present react-ui-components,  i just moved it to ui-classic, D3 3.5.17 is being installed elsewhere?

Reply:
@kavyanekkalapu i am seeing js errors on host summary screen, related to timelines
![Screen Shot 2021-03-23 at 9 18 05 AM](https://user-images.githubusercontent.com/3450808/112152528-c4fb9f00-8bb8-11eb-8b95-0d396dd15727.png)

Reply:
Correct, @pf3/timelines needs older version of D3, i will try to put old version and see, carbon-charts need D3 5.X, but hopefully it will work, thanks for checking @himdel @h-kataria